lib/_codemap.js

```diff
@@ -20,5 +20,7 @@ module.exports = {
   'lrc': require('./lrc'),
   'adx': require('./adx'),
   'vwap': require('./vwap'),
-  'slow_stochastic': require('./slow_stochastic')
+  'slow_stochastic': require('./slow_stochastic'),
+
+  'collection-service': require('./services/collection-service')
```

Comment:
I think, "/services/collection-service'" should be named "require('./services/collection'). No need to duplicate the "service"

Reply:
True, there's no _need_, still I think it adds more than it hurts.